pallets/regulated-assets/Cargo.toml

```diff
@@ -0,0 +1,66 @@
+[package]
+authors = ["Polka Biome Ltd. <jihoon@tutanota.de>"]
+license = "BSD-4-Clause"
+homepage = "https://sora.org"
+repository = "https://github.com/sora-xor/sora2-network"
+edition = "2021"
+name = "regulated-assets"
+version = "1.0.0"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3", default-features = false, features = [
+    "derive",
+] }
+scale-info = { version = "2", default-features = false, features = ["derive"] }
+frame-benchmarking = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false, optional = true }
+frame-support = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
+frame-system = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
+sp-core = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
+sp-std = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38", default-features = false }
+hex-literal = { version = "0.4.1", optional = true }
```

Comment:
used in tests, defined in dev-dependencies
```suggestion
```

Reply:
@Alexey-N-Chernyshov It's used in runtime-benchmarks.. So, I think it's mandatory to add optional dep here and define it at runtime-benchmarking...Am I missing something here?

Reply:
Yes, you are right.